web-common/src/features/dashboards/state-managers/selectors/pivot.ts

```diff
@@ -1,7 +1,47 @@
 import type { DashboardDataSources } from "./types";
+import { PivotChipType } from "../../pivot/types";
 
 export const pivotSelectors = {
   showPivot: ({ dashboard }: DashboardDataSources) => dashboard.pivot.active,
   rows: ({ dashboard }: DashboardDataSources) => dashboard.pivot.rows,
   columns: ({ dashboard }: DashboardDataSources) => dashboard.pivot.columns,
+  measures: ({ metricsSpecQueryResult, dashboard }: DashboardDataSources) => {
+    const measures =
+      metricsSpecQueryResult.data?.measures?.filter(
+        (d) => d.name && dashboard.visibleMeasureKeys.has(d.name),
```

Comment:
Wondering if we should only limit to visible measures and dimensions in the pivot. We don't have controls to change the visibility in the pivot view. Probably better if we show all available dimensions and measures cc @mindspank What do you recon we should do here?

Reply:
@djbarnwal Yes I would display all dimensions and measures